ludwig/schema/utils.py

```diff
@@ -170,14 +186,14 @@ def PositiveInteger(default: Union[None, int] = None, allow_none=False, descript
                 allow_none=allow_none,
                 load_default=default,
                 dump_default=default,
-                metadata={"description": description},
+                metadata={"description": description, **kwargs},
             )
         },
         default=default,
     )
 
 
-def NonNegativeInteger(default: Union[None, int] = None, allow_none=False, description=""):
+def NonNegativeInteger(default: Union[None, int] = None, allow_none=False, description="", **kwargs):
```

Comment:
Could we make some of these parameters required?

`def NonNegativeInteger(default: Union[None, int], allow_none: bool, description: str, **kwargs):`